docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.14
+LABEL maintainer="Quickwit, Inc. <hello@quickwit.io>"
+
+# Install tool
+RUN apk add --no-cache curl
+
+# Create working directory
+RUN mkdir -p /usr/quickwit
```

Comment:
`WORKDIR` will create the directory for you.

Reply:
I believe the Linux convention for installing third-party software is to use the `/opt` folder. However, in this case, since we only copy a binary, let's copy the file directly in `/usr/local/bin`.